src/components/Progress/ProgressBar/style/ProgressBar.style.ts

```diff
@@ -0,0 +1,129 @@
+import styled, { css, keyframes } from 'styled-components';
+import { Text } from '../../../Typography';
+import { Position, Type } from '../../../../../lib';
+
+interface StyledProgressProps {
+    positionText?: Position;
+}
+
+interface StyledBarProps {
+    positionText?: Position;
+}
+
+interface StyledFillProps {
+    type?: Type;
+}
+
+interface StyledFillContainerProps {
+    progress: Readonly<number>;
+}
+
+interface StyledProgressTextProps {
+    positionText?: Position;
+}
```

Comment:
Sjednotil bych všechny rozhraní `interface` do jednoho. Rozumím, že to může být teď pro tebe matoucí ale v Atlanticu je ve zvyku psát jeden interface a dosadit to pak do všech styled component.

Výsledek by pak vypadal zhruba takto:
```
interface StyledProgressBarProps {
   progress: Readonly<number>;
   positionText?: Position;
   type?: Type;
}

export const StyledProgressBar = styled.div<StyledProgressBarProps>...

export const StyledBar = styled.div<StyledProgressBarProps>...

export const StyledFillContainer = styled.div<StyledFillContainerProps>...
```

Můžeš se nechat dát inspirovat třeba zde https://github.com/ThePatriczek/React-Atlantic/blob/master/src/components/Select/Transfer/style/Transfer.style.ts

Reply:
ok